### DISCUSSION TITLE
Feature: Disable access log when using live-reload

### DISCUSSION THREAD

**Discussion by RafaelWO** [tiangolo/uvicorn-gunicorn-docker#233] [Questions] [answered]
When using `gunicorn` (via `start.sh`), one can disable the API access log via setting the corresponding environment variable to an empty value: `ACCESS_LOG=`. However, this is not possible when running the app with `uvicorn` and live-reload (via `start-reload.sh`).

But since you can pass the option `--no-access-log` to `uvicorn` (see [docs](https://www.uvicorn.org/settings/#logging)), the script could be extended to disable the access log if the variable is set to an empty value.

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image

That also means that all the configurations are much simpler as you only need to deal with Uvicorn, not with Gunicorn on top. :sunglasses:
